jvm-packages/xgboost4j-example/src/main/java/ml/dmlc/xgboost4j/java/example/BasicWalkThrough.java

```diff
@@ -46,20 +47,19 @@ public static boolean checkPredicts(float[][] fPredicts, float[][] sPredicts) {
     return true;
   }
 
-
   public static void main(String[] args) throws IOException, XGBoostError {
     // load file from text file, also binary buffer generated by xgboost4j
-    DMatrix trainMat = new DMatrix("../../demo/data/agaricus.txt.train");
-    DMatrix testMat = new DMatrix("../../demo/data/agaricus.txt.test");
+    DMatrix trainMat = new DMatrix("../demo/data/agaricus.txt.train");
+    DMatrix testMat = new DMatrix("../demo/data/agaricus.txt.test");
```

Comment:
it is not necessary to change, as we expect the user to run under spark-examples directory